Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -60,7 +50,7 @@ def __init__(self):
         self.upnp_ports = []
 
         self.session = None
-        self.sesslock = None
+        self.session_lock = None
```

Comment:
As we agreed that the lock can be removed, I'll fix that in the next PR. See https://github.com/Tribler/tribler/issues/2286.